TROUBLESHOOTING.md

```diff
@@ -0,0 +1,28 @@
+![Insights Solution logo](./images/insights-solution.png)
+
+# IRIS client troubleshooting
+
+## Issues when trying to migrate Microsoft Authenticator credentials for IRIS
+
+<details>
+    <summary><h3>"Action required: Scan the QR code provided by your organisation to finish recovering this account"</h3></summary>
+If you encounter this error when attempting to migrate multi-factor authentication to a new device:
+
+1. In an incognito tab visit "https://mysignins.microsoft.com/security-info"
+2. Click "Sign-in options" at the bottom of the Microsoft sign-in page
+
+![Sign-in-options](./images/troubleshooting/sign-in-options.png)
+
+3. From the options, click "Sign in to an organisation"
+4. Type "elexon.co.uk" and click "Next"
+5. In the subsequent sign-in screen, enter the email you/the user signed up with, and click "Next". Continue the sign-in process as normal
```

Comment:
I think `you/the user` can be simplified to `you` since the rest of the instructions is written in that form, e.g. `once you have signed in` (below). I think it reads better to be consistent and, if you're doing it on behalf of someone, just following 'you' as them makes the most sense